faults.md

```diff
@@ -7,10 +7,10 @@ A fault is what happens when partcipants in the protocol are behaving incorrectl
 ### Consensus Faults
 
 - **Duplicate Block Submission Slashing:**
-  - **Condition:** If any miner posts two different blocks for the same chain height.
+  - **Condition:** If any miner posts two blocks satisfying the slashing conditions defined in [Expected Consensus](./expected-consensus.md).
```

Comment:
Attempting to avoid duplication across files.

Reply:
should the reference point be ec spec or faults spec @whyrusleeping 

I can quickly see the faults spec becoming a series of links to other docs (which might be fine)